apps/etterlatte-brev-api/src/main/kotlin/no/nav/etterlatte/brev/notat/NotatRoute.kt

```diff
@@ -0,0 +1,159 @@
+package no.nav.etterlatte.brev.notat
+
+import com.fasterxml.jackson.annotation.JsonTypeInfo
+import com.fasterxml.jackson.annotation.JsonTypeName
+import io.ktor.http.HttpStatusCode
+import io.ktor.server.application.ApplicationCall
+import io.ktor.server.application.call
+import io.ktor.server.request.receive
+import io.ktor.server.response.respond
+import io.ktor.server.routing.Route
+import io.ktor.server.routing.delete
+import io.ktor.server.routing.get
+import io.ktor.server.routing.post
+import io.ktor.server.routing.route
+import io.ktor.util.pipeline.PipelineContext
+import no.nav.etterlatte.brev.EtterlatteBrevKode
+import no.nav.etterlatte.brev.NotatService
+import no.nav.etterlatte.brev.NyNotatService
+import no.nav.etterlatte.brev.model.Slate
+import no.nav.etterlatte.brev.model.Spraak
+import no.nav.etterlatte.libs.common.behandling.BrevbakerBlankettDTO
+import no.nav.etterlatte.libs.common.behandling.Klage
+import no.nav.etterlatte.libs.common.feilhaandtering.UgyldigForespoerselException
+import no.nav.etterlatte.libs.common.sak.Sak
+import no.nav.etterlatte.libs.ktor.brukerTokenInfo
+import no.nav.etterlatte.libs.ktor.route.SAKID_CALL_PARAMETER
+import no.nav.etterlatte.libs.ktor.route.Tilgangssjekker
+import no.nav.etterlatte.libs.ktor.route.medBody
+import no.nav.etterlatte.libs.ktor.route.withSakId
+import org.slf4j.LoggerFactory
+import java.util.UUID
+
+@JsonTypeInfo(use = JsonTypeInfo.Id.NAME, include = JsonTypeInfo.As.PROPERTY, property = "type")
+sealed class StrukturertBrev {
```

Comment:
Kan kanskje flytte denne ut til et eget notat domene?

Reply:
Tilhører notat-greier Øyvind satt opp for en tid tilbake. Siden det skal skrotes gidder jeg ikke brukes særlig tid på opprydding her.